ironfish/src/metrics/meter.ts

```diff
@@ -31,15 +38,17 @@ export class Meter {
   private _intervalMs: number
   private _intervalLastMs: number | null = null
 
-  constructor() {
+  constructor(options?: newMeterOptions) {
```

Comment:
Can we in-line this type? It makes people have to look at a separate interview just for parameters of a function

Reply:
the options type is also used upstream, in `MetricsMonitor.addMeter()` should it be in-lined in both places?